CHANGES.txt

```diff
@@ -1,3 +1,12 @@
+1.4.0 (September 28, 2021)
+ - Updated @splitsoftware/splitio dependency to version 10.16.0, which includes:
+      - Updated localhost mode to emit SDK_READY_FROM_CACHE event in Browser when using localStorage (issue https://github.com/splitio/react-client/issues/34).
+      - Updated streaming logic to use the newest version of our streaming service, including:
+            - Integration with Auth service V2, connecting to the new channels and applying the received connection delay.
+            - Implemented handling of the new MySegmentsV2 notification types (SegmentRemoval, KeyList, Bounded and Unbounded)
+            - New control notification for environment scoped streaming reset.
```

Comment:
This is different than what we were doing (mostly a short of the main changes on the release) but I'm OK with it if it's a pattern we will follow, I like it. Let's only list new functionality or fixes that are important and noticeable here though, to avoid having too big of a list if there's many changes on the upstream Split deps.